doc/source/serve/getting_started.md

```diff
@@ -14,48 +25,43 @@ If you have suggestions on how to improve this tutorial,
 To run this example, you will need to install the following:
 
 ```bash
-$ pip install "ray[serve]" transformers
+$ pip install "ray[serve]" transformers requests
 ```
 
 
 ## Example Model
 
-Let's first take a look at how the model works without using Ray Serve.
-This is the code for the model:
+First, let's take a look at our text-translation model. Here's it's code:
 
-```{literalinclude} ../../../python/ray/serve/examples/doc/e2e_local.py
-:end-before: __local_model_end__
+```{literalinclude} ../serve/doc_code/getting_started/models.py
+:start-after: __start_translation_model__
+:end-before: __end_translation_model__
 :language: python
 :linenos: true
```

Comment:
Is there an easy way to make the line numbers not get copied when using this button?

<img width="878" alt="Screen Shot 2022-07-29 at 2 36 43 PM" src="https://user-images.githubusercontent.com/5459654/181846281-0b15e19d-933e-42dc-afd4-182cac0bc2d0.png">

Reply:
Unfortunately not– this is a known issue: https://github.com/ray-project/ray/issues/21968.

Users can highlight and copy the text without copying numbers, but the clipboard will pick up the numbers. For consistency, I'd prefer to keep these numbers instead of using commented references, but I'm open to changing them.

Reply:
Cool, keeping it this way sounds good